### DISCUSSION TITLE
Help, integrating new features to personal repo forked a long time a go

### DISCUSSION THREAD

**Discussion by David-Morales-Norato** [alshedivat/al-folio#2340] [Q&A] [answered]
Hi, I dont have a lot of experience working with github and using public repos.

I want to integrate new features that have been included in this main repository, such as Google Scholar statistics and image slider integrations for posts.

I forked and customized this repo a few months ago but a lot of new commits have been made. I dont know how to track the specific code I need to integrate this features. 

Anyone have any advise?

**Comment by george-gca** [comment, marked as answer]
Similar to #2320.
